### PR TITLE
image: add ffbs-collect-debug-info

### PR DESCRIPTION
**Pull request by T0biii**
https://github.com/freifunk-gluon/community-packages/pull/130

### CODE REVIEW

File: image-customization.lua [hunk 1]
```diff
@@ -22,6 +22,7 @@ packages{
     'ffac-autoupdater-wifi-fallback',
     'ffac-change-autoupdater',
     'ffac-ssid-changer',
+    'ffbs-collect-debug-info'
```
Comment (grische) [inline image-customization.lua]:
```suggestion
    'ffbs-collect-debug-info',
```

Comment (T0biii) [reply image-customization.lua]: upsi resolved, thanks grische

Comment (T0biii) [reply image-customization.lua]: can we run a linter or something like black? to notice this silly mistakes? 
@grische

Comment (grische) [reply image-customization.lua]: You can try to add this linter here (not the whole file):
https://github.com/freifunk-gluon/gluon/blob/c5c993582b0e302b6c694e0bda74dc7735c30337/.github/workflows/lint.yml#L20

**Comment by T0biii** [comment]
works an my test vm (console gets a lot of output) :) 
so LGTM

**Review by grische — APPROVED** [review]

**Pull request merged**